src/styles/_ui-select.less

```diff
@@ -0,0 +1,75 @@
+/* Fix some styles with the angular-ui-select bootstrap theme that isn't right when combined with all of our styles */
+.ui-select-bootstrap {
+  .surface-shaded & {
```

Comment:
This one `.surface-shaded` style should probably stay in origin-web-console since it's specific to a class that's only defined in origin-web-console.